R/ancova.R

```diff
@@ -1,18 +1,49 @@
 
 #' Analysis of Covariance
 #'
-#' Performs an analysis of covariance between two groups returning the estimated "treatment effect"
-#' as well as the least squared means of each group.
+#' Performs an analysis of covariance between two groups returning the estimated
+#' "treatment effect" and least squared means estimates of each group.
 #'
-#' @param data A dataframe containing the variables to be used in the model
-#' @param vars A named list containing the names of the variables for `group`, `visit`, `outcome` and any `covariates`.
+#' @param data A data.frame containing the data to be used in the model
+#' @param vars A vars object as generated by [set_vars()]. Only the `group`,
+#' `visit`, `outcome` and `covariates` elements are required. See details.
+#' @param visits an optional character vector specifying which visits to
+#' fit the ancova model at. If `NULL` all available
+#' visits (as determined by `unique(data[[vars$visit]])`) will be used.
 #' See details.
-#' @param visits an optional character vector specifying which visits to perform ancova at. If `NULL` all all available
-#' visits (as determined by `unique(data[[vars$visit]])`) will be looped over.
 #'
 #' @details
-#' TODO
+#' The function works as follows
 #'
+#' 1. Select the first value from `visits`
+#' 2. Subset the data to only the observations that succoured on this visit
```

Comment:
Is "soccoured" correct sorry? (My english vocabulary is a bit limited)

Reply:
no, no it is not correct :) 

Updated..